lib/ramble/ramble/cmd/workspace.py

```diff
@@ -587,6 +589,10 @@ def workspace_info(args):
 
                 for exp_name, _, _ in print_experiment_set.filtered_experiments(filters):
                     app_inst = experiment_set.get_experiment(exp_name)
+                    if app_inst.uses_spack:
```

Comment:
Not important for this PR, but I guess in the future we should abstract out the package manager and we more like:

```
if app_inst.package_manager == PACKAGE_MANAGERS.SPACK:
  ...
```

edit: Oh you even just added  a `package_managers` below in `software_environments` :)

Reply:
Yeah, I think we need to consider how to make the package manager stuff better though. So I didn't go too crazy into trying to design something that would work in a general sense.